src/components/card/card.module.css

```diff
@@ -7,8 +7,12 @@
 	/* used by components like CardLink */
 	--card-border-radius: 6px;
 
-	background-color: white;
+	background-color: var(--token-color-surface-primary);
```

Comment:
In 'light' mode, [this is the same as white or #fff](https://helios.hashicorp.design/foundations/colors?tab=palette#surface-1)